code/modules/trade/datums/trade_stations_presets/asterstradecapital.dm

```diff
@@ -102,11 +102,52 @@
 			/obj/item/organ_module/active/simple/armshield
 		),
 	)
+	secret_inventory = list(
+		"Exosuits" = list(
+			/mob/living/exosuit/premade/powerloader/firefighter,
+			/mob/living/exosuit/premade/powerloader/flames_blue,
+			/mob/living/exosuit/premade/powerloader/flames_red,
+			/mob/living/exosuit/premade/light,
+			/mob/living/exosuit/premade/heavy,
+			/mob/living/exosuit/premade/combat/slayer,
+		),
+		"Mech Armor" = list(
+			/obj/item/robot_parts/robot_component/armour/exosuit/plain,
+			/obj/item/robot_parts/robot_component/armour/exosuit/radproof,
+			/obj/item/robot_parts/robot_component/armour/exosuit/ablative,
+			/obj/item/robot_parts/robot_component/armour/exosuit/combat,
+		),
+		"Parts" = list(
+			/obj/item/mech_component/chassis,
+			/obj/item/mech_component/manipulators,
+			/obj/item/mech_component/sensors,
+			/obj/item/mech_component/propulsion,
+		),
+		"Soft" = list(
+			/obj/item/electronics/circuitboard/exosystem/engineering,
+			/obj/item/electronics/circuitboard/exosystem/utility,
+			/obj/item/electronics/circuitboard/exosystem/medical,
+			/obj/item/electronics/circuitboard/exosystem/weapons,
+		),
+		"Equipment" = list(
+			/obj/item/mech_equipment/mounted_system/taser,
+			/obj/item/mech_equipment/mounted_system/taser/ion,
+			/obj/item/mech_equipment/mounted_system/taser/plasma,
+			/obj/item/mech_equipment/mounted_system/rcd,
+			/obj/item/mech_equipment/clamp,
+			/obj/item/mech_equipment/light,
+			/obj/item/mech_equipment/drill,
+			/obj/item/mech_equipment/mounted_system/extinguisher,
+			/obj/item/mech_equipment/sleeper,
+		),
+	)
 	offer_types = list(
-		/obj/item/mech_component/ = offer_data("mech component", 125, 10),															// base price: 150
-		/obj/item/mech_equipment/ = offer_data("mech equipment", 175, 10),															// base price: 200
-		/obj/item/robot_parts/robot_component/armour/exosuit/plain = offer_data("exosuit armor plating", 250, 8),					// base price: 300, sold at common
-		/obj/item/robot_parts/robot_component/armour/exosuit/radproof = offer_data("rad-proof exosuit armor plating", 415, 8),		// base price: 500, sold at common
-		/obj/item/robot_parts/robot_component/armour/exosuit/ablative = offer_data("ablative exosuit armor plating", 465, 8),		// base price: 550, sold at common
-		/obj/item/robot_parts/robot_component/armour/exosuit/combat = offer_data("combat exosuit armor plating", 830, 8),			// base price: 1000, sold at common
+		/obj/item/mech_component/ = offer_data("mech component", 150, 10),															// base price: 150
```

Comment:
```suggestion
		/obj/item/mech_component/ = offer_data("mech component", 150, 10),															// base price: 150
```
Not sure if this `/` should stay after mech component. Maybe it does not change anything